templates/3/conf.d/listen.erb

```diff
@@ -0,0 +1,31 @@
+listen {
+  type = <%= @listen_type %>
+<% if [ 'ALL', 'any' ].include? @ipaddr -%>
+  ipaddr = *
+<% elsif @ipaddr.include?(':') -%>
+  ipv6addr = <%= @ipaddr %>
+<% else -%>
+  ipaddr = <%= @ipaddr %>
+<% end -%>
+  port = <%= @port %>
+<% if @interface -%>
+  interface = <%= @interface %>
+<% end -%>
+<% if @per_socket_clients -%>
+  clients = <%= @per_socket_clients %>
+<% end -%>
+  limit {
+<% if @max_pps -%>
```

Comment:
should this be max_epps?

Reply:
Fixed.  It should have been max_pps in the manifest.